sdk/CoreWCF/Azure.CoreWCF.AzureQueueStorage/tests/IntegrationTests.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Contracts;
+using CoreWCF.AzureQueueStorage.Tests.Helpers;
+using Microsoft.AspNetCore.Hosting;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace CoreWCF.AzureQueueStorage.Tests
+{
+    public class IntegrationTests
+    {
+        private readonly ITestOutputHelper _output = null;
+        public const string QueueName = "AzureQueue";
+
+        [Fact(Skip = "Need AzureQueue")]
```

Comment:
We'll want to convert this over to the Azure SDK test infrastructure for live test support.